### PR TITLE
Re-using recommendation#index style for recommendation#show

### PR DESCRIPTION
**Pull request by bumi**
This makes the two pages look similar and reuses the same structure.

### CODE REVIEW

**Comment by bumi** [comment]
What do you think about this?

**Comment by musayann** [comment]
Where then should we put, the back to recommendations button ( or breadcrumb )

File: app/javascript/views/recommendations/Show.vue [hunk 1]
```diff
@@ -1,33 +1,48 @@
 <template>
   <!-- eslint-disable vue/no-v-html -->
+  <div class="w-100 mx-auto bg-white min-h-screen -mt-8">
+    <div class="relative pt-5">
+      <div class="w-80 mx-auto  h-20 bg-transparent border-grey-light shadow-sm border-solid border-1 ">
+        <h1 class="text-primary text-2xl pt-4 my-2 pl-5 xs:pl-0 xs:px-3 xs:text-xl xs:pt-2 md:text-xl md:pl-0 md:px-5 md:pt-2">
+          Our recommendations for your software development best practices.
```
Comment (musayann) [inline app/javascript/views/recommendations/Show.vue]: Shouldn't we display the current recommendation title here instead

Comment (bumi) [reply app/javascript/views/recommendations/Show.vue]: hmm, we can try that...

File: app/javascript/views/recommendations/Show.vue [hunk 2]
```diff
@@ -1,33 +1,42 @@
 <template>
   <!-- eslint-disable vue/no-v-html -->
+  <div class="w-85 mx-auto bg-white min-h-screen overflow-hidden my-4 rounded-lg py-5">
+    <div class="relative pt-4">
+      <div class="w-85 mx-auto  h-20 bg-transparent border-grey-light shadow-sm border-solid border-1 overflow-hidden">
+        <h1 class="text-primary text-2xl  my-4 pl-5 xs:pl-0 xs:px-3 xs:text-xl xs:pt-2 md:text-xl md:pl-0 md:px-5 md:pt-2">
+         {{ recommendation.title }}
```
Comment (bumi) [inline app/javascript/views/recommendations/Show.vue]: how about we add the breadcrumb here? 

`<< Recommendations - Title`

**Pull request merged**